modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/admin/beta_client_extension_button.jsp

```diff
@@ -8,18 +8,8 @@
 <%@ include file="/admin/init.jsp" %>
 
 <div>
-	<react:component
-		module="{FeatureIndicator} from frontend-js-components-web"
-		props='<%=
-			HashMapBuilder.<String, Object>put(
-				"interactive", "true"
-			).put(
-				"learnResourceContext", LearnMessageUtil.getReactDataJSONObject("frontend-js-components-web")
-			).put(
-				"tooltipAlign", "bottom"
-			).put(
-				"type", "beta"
-			).build()
-		%>'
+	<liferay-frontend:feature-indicator
```

Comment:
Hey @pat270 can you remove the extra div at line 10. We shouldn't need this here or in beta_button.jsp since our taglib already includes an extra div for the react component to be rendered into. We do still need the span in view_info_collection_providers.jsp since that keeps it as an inline element.

Could you also include screenshots of each of these locations so that we can guarantee we aren't accidentally breaking anything.